src/azul/types.py

```diff
@@ -1,3 +1,4 @@
+import typing
```

Comment:
We shouldn't combine whole module imports and individual symbol imports from the same module.